server/devdocs/index.js

```diff
@@ -237,7 +239,7 @@ module.exports = function() {
 
 		const fileContents = fs.readFileSync( path, { encoding: 'utf8' } );
 
-		response.send( marked( fileContents ) );
+		response.send( 'html' === format ? marked( fileContents ) : fileContents );
```

Comment:
What behavior were we seeing without this change?

Reply:
`marked` parses the content of a markdown file to html, but we [need the actual markdown](https://github.com/Automattic/wp-calypso/blob/6cc81d883b0ce846e7763910b15eff94b2e6c036/client/devdocs/gutenberg-components/example.jsx#L32) content for [retrieving the example code](https://github.com/Automattic/wp-calypso/blob/6cc81d883b0ce846e7763910b15eff94b2e6c036/client/devdocs/gutenberg-components/example.jsx#L43) from the readme.

Reply:
Thanks for the context!